### DISCUSSION TITLE
[Announment] Welcome new committer: Stephen Ni

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#3731] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that [Stephen Ni](https://github.com/nisiyong) a.k.a nisiyong, has been voted in as a new Apollo committer.

Please join me to say congratulations to him!

@nisiyong, would you please briefly introduce yourself to the community?

Best Regards,
Jason

**Comment by kezhenxu94** [comment]
Congratulations @nisiyong and welcome aboard!!!
